src/libraries/Microsoft.Extensions.Http/src/ExpiredHandlerTrackingEntry.cs

```diff
@@ -30,5 +30,19 @@ public ExpiredHandlerTrackingEntry(ActiveHandlerTrackingEntry other)
         public string Name { get; }
 
         public IServiceScope? Scope { get; }
+
+        public void Dispose()
+        {
+            try
+            {
+                InnerHandler.Dispose();
+                Scope?.Dispose();
+            }
+            catch
+            {
+                // Ignore exceptions during disposal
+                throw;
+            }
```

Comment:
this is redundant

on the side note, I think we'd better dispose the scope even if the handler's disposal throws, so Scope?.Dispose() can go into finally